docs/reference/commandline/events.md

```diff
@@ -111,77 +111,94 @@ You'll need two shells for this example.
 
 **Shell 2: Start and Stop containers:**
 
-    $ docker start 4386fb97867d
-    $ docker stop 4386fb97867d
-    $ docker stop 7805c1d35632
+    $ docker create --name test alpine:latest top
+    $ docker start test
+    $ docker stop test
 
 **Shell 1: (Again .. now showing events):**
 
-    2015-05-12T11:51:30.999999999Z07:00 container start 4386fb97867d (image=ubuntu-1:14.04)
-    2015-05-12T11:51:30.999999999Z07:00 container die 4386fb97867d (image=ubuntu-1:14.04)
-    2015-05-12T15:52:12.999999999Z07:00 container stop 4386fb97867d (image=ubuntu-1:14.04)
-    2015-05-12T15:53:45.999999999Z07:00 container die 7805c1d35632 (image=redis:2.8)
-    2015-05-12T15:54:03.999999999Z07:00 container stop 7805c1d35632 (image=redis:2.8)
+    2017-01-05T00:35:58.859401177+08:00 container create 0fdb48addc82871eb34eb23a847cfd033dedd1a0a37bef2e6d9eb3870fc7ff37 (image=alpine:latest, name=test)
```

Comment:
Thanks for your feedback @thaJeztah
I did some change. And please help me check since I was wondering if I follow your point correctly.  😄